### PR TITLE
Update dockerfile

### PR DESCRIPTION
**Pull request by buehlefs**
Exclude plugins from default docker build.
Allow plugins to be loaded from git into docker container.
Migrate DB on container startup.
Add container startup tasks in tasks.py.

TODO:

- [ ] Documentation for dockerfile
- [ ] Automatic docker build using GH Actions

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,52 @@
-FROM ubuntu:focal
+FROM python:3.9-slim
+
+LABEL org.opencontainers.image.source="hhttps://github.com/UST-QuAntiL/qhana-plugin-runner"
```
Comment (PhilWun) [inline Dockerfile:3]: The url has one "h" too many.

File: Dockerfile [hunk 2]
```diff
@@ -1,15 +1,52 @@
...
+# create and activate virtualenv as gunicorn user to allow installing additional dependencies later
+ARG VIRTUAL_ENV=/venv
+RUN python -m venv ${VIRTUAL_ENV}
+
+# change path to include virtualenv first (affects all following commands)
+ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
+
+RUN python -m pip install --upgrade pip && python -m pip install gunicorn poetry invoke
+
+COPY --chown=gunicorn . /app
```
Comment (PhilWun) [inline Dockerfile:44]: Plugins are still copied from the plugins folder into the Docker Container. Is this intentional?

Comment (buehlefs) [reply Dockerfile:44]: No, plugins should be ignored as per dockerignore file.

File: Dockerfile (hunk 1 above)
Comment (buehlefs) [reply Dockerfile]: Should be fixed in all three repositories now.

**Pull request merged**